examples/aws-tf-ha/main.tf

```diff
@@ -0,0 +1,126 @@
+provider "aws" {
+  region = var.region
+}
+
+module "instances" {
+  source           = "../aws-tf"
+  controller_count = var.controller_count
+  worker_count     = var.worker_count
+}
+
+data "aws_vpc" "default" {
+  default = true
+}
+
+data "aws_subnet_ids" "all" {
+  vpc_id = data.aws_vpc.default.id
+}
+
+module "elb" {
+  source = "terraform-aws-modules/elb/aws"
+
+  name = "elb"
+
+  subnets         = data.aws_subnet_ids.all.ids
+  security_groups = [module.instances.cluster-security_group]
+  internal        = false
+
+  listener = [
+    {
+      instance_port     = "80"
+      instance_protocol = "http"
+      lb_port           = "80"
+      lb_protocol       = "http"
+    },
+    {
+      instance_port     = "8080"
+      instance_protocol = "http"
+      lb_port           = "8080"
+      lb_protocol       = "http"
+    },
```

Comment:
Why do we need port 80 & 8080? We've got nothing on controllers listening on those